src/test/ui/consts/const-needs_drop-monomorphic.stderr

```diff
@@ -0,0 +1,20 @@
+error[E0599]: no function or associated item named `assert` found for struct `Bool<{ std::mem::needs_drop::<T>() }>` in the current scope
```

Comment:
This error is useless - it would be good to suppress it. Is there already an issue for this kind of error?